java/third_party/workspace.bzl

```diff
@@ -0,0 +1,73 @@
+# Do not edit. bazel-deps autogenerates this file from java/dependencies.yaml.
```

Comment:
Does this need to be checked in? Generating this on the fly would be better.

Reply:
When generating this file with  java/generate_deps.sh(command: bazel run), it should be in the path as well, otherwise it will report an error because it is referenced in WORKSPACE file.

Reply:
And the return content of the list_dependencies funtion in this file is empty which will be autogenerate by  java/generate_deps.sh.